src/client.rs

```diff
@@ -88,6 +88,34 @@ impl DatabaseClient for GenericClient {
     }
 }
 
+pub struct Config {
+    pub url: url::Url,
+    pub token: Option<String>,
```

Comment:
`auth_token` to match TS?

https://github.com/libsql/libsql-client-ts/pull/10/commits/deab5f21efd673604433ed0ccb9a7dd544e4d7dd